### DISCUSSION TITLE
NumbaPerformanceWarning: Grid size 16 will likely result in GPU under-utilization due to low occupancy

### DISCUSSION THREAD

**Discussion by mehadi92** [NVIDIA-NeMo/NeMo#6820] [Q&A] [answered]
Facing this during conformer transducer ASR model training. Should I ignore this? I'm using `nvcr.io/nvidia/nemo:23.03`

**Comment by titu1994** [comment, marked as answer]
This is ok to ignore. It's just a performance warning and not an error. The cuda kernel is not large enough to fill the GPU memory due to RNNT memory cost.
